docs/dasharo-naming-convention.md

```diff
@@ -0,0 +1,79 @@
+# Dasharo Product Naming Convention
+
+Following documentation is results of [RFC](https://github.com/Dasharo/dasharo-issues/issues/762).
+
+## Synopsis
+
+```plain
+Dasharo (firmware_framework[+payload]) Entry Package [customer_type] for
+market_segment
+```
+
+## Description
+
+Dasharo's naming scheme is crafted to convey the essential details of each
+firmware package. It includes the base firmware framework, an optional payload,
+the targeted market segment, and the customer type. This structure assists in
+identifying the most suitable package for specific technological needs, market
+segments, and customer categories.
+
+Components of the naming scheme:
+
+```plain
+Dasharo (firmware_framework[+payload]) Entry Package [customer_type] for
+market_segment
```

Comment:
@pietrushnic I'm thinking about using the following scheme:

```suggestion
Dasharo (firmware_framework[+payload]) [customer_type] Package for
market_segment
```
This way, we avoid using the words `Entry` and `Pro` in the same product name, which, in my opinion, are opposites. Also, it shortens the full name a little.

Reply:
This is a very good idea. Implemented: 4473a377a4d6

Now we have DPP and DEP; the second has a collision with the Dasharo Cooperation Model called the Dasharo Enterprise Package. I have to improve the naming for the second one.

Reply:
@artur-rs ?

Reply:
@pietrushnic resolving the thread